backend/migrations/000014_add_one_address_strategy.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS voting_strategies;
```

Comment:
We shouldn't drop the entire table in the down. Instead this should just undo whatever is defined in the up.
Instead this should be:
```
DELETE FROM voting_strategies WHERE key='one-address-one-vote' LIMIT 1
```

Reply:
got it. Thanks @dbslone, new to these migrations.